railties/lib/rails/commands/app/update_command.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require "rails/generators"
+require "rails/generators/rails/app/app_generator"
+
+module Rails
+  module Command
+    module App
+      class UpdateCommand < Base # :nodoc:
+        class_option :force, aliases: "-f", type: :boolean, default: false, desc: "Overwrite all files"
```

Comment:
Instead of this we could do:
```suggestion
        include Thor::Actions
        add_runtime_options!
```

to support all the options that Thor supports, like `--skip`, `--pretend` etc.

Reply:
Let's make this a Thor::Actions